mne/decoding/transformer.py

```diff
@@ -168,35 +169,41 @@ def __init__(self, info=None):
         self.n_channels = None
         self.n_times = None
 
-    def fit(self, epochs_data, y):
+    def fit(self, epochs, y=None):
         """For each epoch, concatenate data from different channels into a single
         feature vector.
 
         Parameters
         ----------
-        epochs_data : array, shape (n_epochs, n_channels, n_times)
+        epochs : instance of Epochs
             The data to concatenate channels.
-        y : array, shape (n_epochs,)
-            The label for each epoch.
+        y : None | array of shape(n_epochs,)
+            The label of each epoch
```

Comment:
epoch.

(Note that y can also be multidimensional: e.g. http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LinearRegression.html#sklearn.linear_model.LinearRegression.fit)